src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx

```diff
@@ -315,7 +315,7 @@ export default function DownloadDialog({
     gameInstallInfo?.game?.owned_dlc?.length > 0
   const DLCList = gameInstallInfo?.game?.owned_dlc
   const downloadSize = () => {
-    if (gameInstallInfo?.manifest?.download_size) {
+    if (gameInstallInfo?.manifest?.download_size !== undefined) {
```

Comment:
Is this to accept a zero as the size since browser games won't have it?

Reply:
updated now :)